requirements.txt

```diff
@@ -1,3 +1,5 @@
-gunicorn
-eventlet==0.30.2
-greenlet>=0.4.14
\ No newline at end of file
+https://github.com/benoitc/gunicorn/archive/refs/heads/master.zip#egg=gunicorn==20.1.0
+https://github.com/eventlet/eventlet/archive/master.zip
```

Comment:
I think we can remove greenlet/eventlet from the requirements if they are not used

Reply:
used guincorn standerd package and removed greenlet/eventlet